docs/tutorials/comparison.md

```diff
@@ -79,302 +123,753 @@ source1 = "FirstSourceName"
+
+2. Set your parameters. The following parameters should be filled:
+   ![VSPreview comp bar](/static/comparison/vsp-comp.png)
+
+   Key | Description
+   ----|---------------------------------------------------------
+   1   | The title of your comparison/show
+   4   | The [TMDB ID](https://www.themoviedb.org) for the show
+
+   - *See [Comp](#comp) for more detail on what each one is used for*
+
+3. Hit the *Start Upload* button under *Comp* to begin creating your comparison
+
+==- Manual
+
+Manual comparison are created completely by the user. VSPreview displays and handles frame capture, while the main actions are performed manually by the user through the previewer.
+
+#### Capturing
+
+1. Locate the frame(s) you want to compare
+   - By default, `Left arrow` and `Right arrow` navigates between frames
+   - *See [tips](#tips) on what you should look for*
+
+2. Once you find a frame, take a screenshot of the current frame
+   - Default keybind: `Shift` + `S`
+
+3. Switch to the other sources and take screenshots of their current frame
+   - Press the number keys to change sources (e.g. `1` for `clip1`, `2` for `clip2`) and press the screenshot keybind
+   - *You may need to readjust the position using the arrow keys*
+
+4. Repeat process for the next frames in your comparison
+
+!!!
+By default, all frames are stored within your working directory unless manually changed to a different destination.
+!!!
+
+!!!secondary
+*See [Post-processing](#post-processing) for additional scripts to apply after creating your comparison.*
+!!!
 
 ==-
 
+## QoL Changes
+
+### VSPreview
+
+==- Recommended source naming
+
+You can name your sources in any way you'd like. However, we recommend trying to follow any of the naming schemes below for better clarity:
+
+![Example source names](/static/comparison/example-sources.png)
+
+Key                                                        | Meaning
+-----------------------------------------------------------|---------------------------------------------
+![#ed8796](https://placehold.co/14x14/ed8796/ed8796.png) 1 | The name of the source/release group
+![#f5a97f](https://placehold.co/14x14/f5a97f/f5a97f.png) 2 | The video resolution
+![#eed49f](https://placehold.co/14x14/eed49f/eed49f.png) 3 | The video codec
+![#a6da95](https://placehold.co/14x14/a6da95/a6da95.png) 4 | The file size
+![#91d7e3](https://placehold.co/14x14/91d7e3/91d7e3.png) 5 | Additional tags, such as scaling or HDR/DV
+
+Depending on what you are trying to compare will depend on how you should name your sources. For example, if you are trying to compare HDR vs. SDR sources, you should include the type in the source name. *Generally, the first three will cover most comparisons you make, but you are free to include more as needed.*
+
+==- Change frame increment
+
+- In VSPreview, navigate to the bottom bar and toggle the *Playback* section
+- In *Playback*, set the number of frames to skip when scrubbing by adjusting the first field:
```

Comment:
this doesn't mention that you need to SHIFT+arrow key. People will assume this will apply to arrow key otherwise